recipe/meta.yaml

```diff
@@ -33,15 +33,6 @@ build:
 
 requirements:
   build:
-    - binutils_impl_{{ target_platform }}  {{ binutils_version }}
-    - binutils_impl_{{ cross_target_platform }}  {{ binutils_version }}
-    - gcc_impl_{{ target_platform }}  {{ gcc_version }}               # [build_platform != target_platform]
-    - gxx_impl_{{ target_platform }}  {{ gcc_version }}               # [build_platform != target_platform]
```

Comment:
Now that we're not requiring the exact same `gcc_version` for cross-compiling anymore, that should also mean that we don't have to merge with failing CI for new GCC versions, which is great 🥳